src/OpalCompiler-Core/OpalCompiler.class.st

```diff
@@ -325,7 +325,6 @@ OpalCompiler >> compile [
 	| cm |
 	[ 	[	self parse.
 			self transformDoit.
```

Comment:
Now #doSemanticAnalysis will be performed two times: in #parse and in #transformDoit. Or I may miss something

Reply:
I have now change the implementation to have a different explicit code path when parsing/compiling doIts, this solves that problem